bin/node/runtime/src/kovan.rs

```diff
@@ -131,6 +134,25 @@ impl BridgePruningStrategy for PruningStrategy {
 	}
 }
 
+/// The Kovan Blockchain as seen by the runtime.
+pub struct KovanBlockchain;
+
+impl PeerBlockchain for KovanBlockchain {
```

Comment:
This implementation should not be necessary, and hopefully will be addressed by #205